include/util/AppendVariant.h

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) TrajoptLib contributors
+
+#pragma once
+
+#include <variant>
+
+// NOLINTBEGIN
+template <typename... Vs, typename... Aps>
+std::variant<Vs..., Aps...> _append_variant(std::variant<Vs...> myvar,
```

Comment:
You could name this variant_cat like https://en.cppreference.com/w/cpp/utility/tuple/tuple_cat.

Reply:
I don't think this function is similar enough to that one. Unlike a tuple, a variant only holds one value, so it doesn't really make sense to make a new one that concatenates them.